lib/favourites/favourites.dart

```diff
@@ -0,0 +1,26 @@
+import 'package:SquanchyFlutter/bottomnavigation/bottomnav.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+class FavouritesView extends StatelessWidget {
+  @override
+  Widget build(BuildContext context)
+  {
+    final theme = Theme.of(context);
+
+    var appBar = new AppBar(
+        iconTheme: theme.iconTheme,
+        title: const Text('FAVOURITES'),
+        backgroundColor: Colors.blue,
+        actions: <Widget>[
+          new SettingsButton(),
+          new SearchButton()
+        ]
+    );
+
+    return new Scaffold(
+      appBar: appBar,
+      backgroundColor: const Color(0xFFF1F1F1),
```

Comment:
Couldn't this be in the theme? Same in the other pages

Reply:
Yeah I realised how to use themes properly at the end of this set of work 😂 will add now